### PR TITLE
Crash when loading sample mbtiles file as offline layer

### PR DESCRIPTION
**Pull request by grzesiek2010**

### CODE REVIEW

**Comment by lognaturel** [comment]
@grzesiek2010 It is nice not to have a crash but I think it's pretty confusing not to have any feedback that the file failed and why it failed! I want to think about this one just a little bit more.

Other feedback appreciated!

**Comment by grzesiek2010** [comment]
@lognaturel I've added a toast please review again.

File: collect_app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -508,4 +508,5 @@
     <string name="type">Type</string>
     <string name="form_entry">Uncheck to hide from Form Entry</string>
     <string name="use_device_language">Use device language</string>
+    <string name="not_supported_offline_layer_format">Selected offline layer file uses PBF format which is not supported!</string>
```
Comment (lognaturel) [inline collect_app/src/main/res/values/strings.xml]: This should be "uses the PBF format" (with "the"). 😊

File: collect_app/src/main/java/org/odk/collect/android/spatial/MapHelper.java [hunk 2]
```diff
@@ -256,5 +262,19 @@ public boolean accept(File dir, String filename) {
         });
     }
 
+    private boolean isFileFormatSupported(File file) {
```
Comment (lognaturel) [inline collect_app/src/main/java/org/odk/collect/android/spatial/MapHelper.java]: Can you please add a little comment here so we remember why this method is here more easily? Might just be "osmdroid doesn't currently support pbf tiles: https://github.com/osmdroid/osmdroid/issues/101"

**Review by lognaturel — CHANGES_REQUESTED** [review]
Couple of small internal changes.

@mmarciniak90 the functionality is there and ready for you to review.

**Comment by grzesiek2010** [comment]
@lognaturel done.

**Review by lognaturel — APPROVED** [review]

**Comment by mmarciniak90** [comment]
Tested with success. 
Verified on Android v4.1, v4.2, v4.4, v5.1, v6.0.
ODK does not crash when I load layer from [Example1](https://www.mapbox.com/help/define-mbtiles/) or [Example2](https://openmaptiles.org/downloads/#country). 
Toast with message 'Selected offline layer file uses the PBF format which is not supported!' is visible
screenshot_20170626-115308
.

**Comment by mmarciniak90** [comment]
@opendatakit-bot unlabel "needs testing"

**Pull request merged**